src/pytest_plugins/forks/forks.py

```diff
@@ -10,9 +10,19 @@
 import pytest
 from pytest import Metafunc
 
-from ethereum_test_forks import (
+from ethereum_test_forks import (  # noqa: F401
+    Berlin,
+    Byzantium,
+    Cancun,
+    Constantinople,
```

Comment:
I would like to find a better way to this, but its needed for the `globals()[fork]` below